src/azure-cli/azure/cli/command_modules/resource/_bicep.py

```diff
@@ -177,6 +187,36 @@ def supports_bicep_publish():
     return semver.compare(installed_version, "0.4.1008") >= 0
 
 
+def _is_running_in_ci():
+    if "GITHUB_ACTIONS" in os.environ or "TF_BUILD" in os.environ:
```

Comment:
Is there a possibility that these vars will be set but Bicep isn't on the PATH? E.g. if using a non-standard container image.